### ISSUE TITLE
Crop method throws

### ISSUE THREAD

**Issue by shaungrady** [STDSuperman/super-image-cropper#25]
Last working version was `1.0.18`, but after that version, the following call throws:

```ts
const superCropper = new SuperImageCropper()

superCropper.crop({
	cropperInstance: cropper, // ReacterCropper instance
	src: 'https://placehold.co/600x400.png',
	cropperJsOpts: { background: '#fff' },
	gifJsOptions: { transparent: null },
	outputType: 'base64',
	crossOrigin: 'anonymous',
})
```

Here's a code playground (https://playcode.io/2037874), click the crop button and see the error in the console. 

```
error {
  ​bubbles: false
  ​cancelBubble: false
  ​cancelable: false
  ​composed: false
  ​currentTarget: null
  ​defaultPrevented: false
  ​eventPhase: 0
  ​explicitOriginalTarget: <img crossorigin="anonymous" src="">
  ​isTrusted: true
  ​originalTarget: <img crossorigin="anonymous" src="">
  ​returnValue: true
  ​srcElement: <img crossorigin="anonymous" src="">
  ​target: <img crossorigin="anonymous" src="">
  ​timeStamp: 6264
  ​type: "error"
  ​<get isTrusted()>: function isTrusted()
​}
```

**Comment by STDSuperman** [comment]
I'll take a look at this issue.

**Comment by STDSuperman** [comment]
The bug has been fixed, and the fix version is 1.0.22.
